lib/state_machines/machine.rb

```diff
@@ -658,6 +658,12 @@ def dynamic_initial_state?
     #   directly to the object
     def initialize_state(object, options = {})
       state = initial_state(object)
+
+      unless object.respond_to? attribute
+        object.class.send(:alias_method, attribute, :state)
```

Comment:
I think we can't hardcode :state . What about custom state column  ? or multiple state machines in the same object ?